src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Backup/BackupStep.cs

```diff
@@ -13,12 +13,13 @@ namespace Microsoft.DotNet.UpgradeAssistant.Steps.Backup
     public class BackupStep : UpgradeStep
     {
         private const string FlagFileName = "upgrade.backup";
+        private const string BackupPropertyName = "BackupLocation";
 
         private readonly bool _skipBackup;
         private readonly IUserInput _userInput;
 
         private string? _projectDir;
-        private string? _backupPath;
+        private string? _defaultBackupPath;
```

Comment:
Can you remove this and just grab it from the context object?

Reply:
Makes sense. I think the same treatment should be applied to _projectDir as well.